### PR TITLE
Add CREATE LIBRARY support

### LINKED ISSUE #124: Feature request: CREATE LIBRARY command
https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_LIBRARY.html

Would be nice for this to be supported in sqlalchemy-redshift.

### PR DESCRIPTION
**Pull request by dargueta**
Closes #124

## Todos
- [x] MIT compatible
- [x] Tests
- [x] Documentation
- [x] Updated CHANGES.rst

### CODE REVIEW

**Comment by dargueta** [comment]
@jklukas sorry to bother you, just wanted to let you know this was here.

**Comment by jklukas** [comment]
Thanks for the ping. I'm on vacation but will take a look next week.

File: tests/test_create_library.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+from sqlalchemy_redshift import commands
+from rs_sqla_test_utils.utils import clean, compile_query
+
+ACCESS_KEY_ID = 'IO1IWSZL5YRFM3BEW256'
+SECRET_ACCESS_KEY = 'A1Crw8=nJwEq+9SCgnwpYbqVSCnfB0cakn=lx4M1'
```
Comment (jklukas) [inline tests/test_create_library.py]: Did you intend for these keys to get committed in this PR?

Comment (jklukas) [reply tests/test_create_library.py]: I see now these are copied from another test as introduced in https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/21

I hadn't been aware that these creds were here and don't quite understand the history of how they got there. Perhaps @graingert remembers more? Was this intended?

Comment (graingert) [reply tests/test_create_library.py]: I'm pretty sure they're inert keys to test the regex

Comment (graingert) [reply tests/test_create_library.py]: Might be worth making it more obvious that they are example creds

Comment (dargueta) [reply tests/test_create_library.py]: Oh yeah sorry about that. Will fix it.

Comment (dargueta) [reply tests/test_create_library.py]: Okay, added a comment.

**Comment by jklukas** [comment]
@dargueta - Can you rebase this on master?

**Comment by dargueta** [comment]
Yep! Done.

**Pull request merged**